build_tools/github_actions/amdgpu_family_matrix.py

```diff
@@ -27,7 +27,7 @@
 amdgpu_family_info_matrix_postsubmit = {
     "gfx950": {
         "linux": {
-            "test-runs-on": "linux-mi355-1gpu-ossci-rocm",
+            "test-runs-on": "",  # removed due to failing sanity check, label is "linux-mi355-1gpu-ossci-rocm"
```

Comment:
What's the difference between this and moving to `amdgpu_family_matrix_xfail`, @geomin12 ?

xfail is only _build_ failures while here would be _test_ failures?

Reply:
I think xfail moves tests to the nightly CI instead of running the builds as part of the post-submit CI but @geomin12 please confirm.